Resources/Textures/Structures/Machines/tabletop_parts.rsi/meta.json

```
@@ -0,0 +1,31 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/0d9c9a8233dfc3fc55edc538955a761a6328bee0, computer frames modified by JustUser",
```

These are not your sprites. Please attribute the original authors properly as per the license terms of the CC-BY-SA.
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/bd6873fd4dd6a61d7e46f1d75cd4d90f64c40894. | Resprited by erhardsteinhauer (discord)",
```